lte/gateway/docker/services/s1aptester/Dockerfile

```diff
@@ -0,0 +1,123 @@
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+ARG OS_DIST=ubuntu
+ARG OS_RELEASE=focal
+
+
+FROM $OS_DIST:$OS_RELEASE
+ARG MAGMA_RELEASE=master
```

Comment:
The described way of how to build this dockerfile does not allow to change the ARGs. Why is this cloning from git instead of building the existing source tree (similar to other dockerfiles)